be/src/common/daemon.cpp

```diff
@@ -408,20 +408,20 @@ void Daemon::start() {
     st = Thread::create(
             "Daemon", "tcmalloc_gc_thread", [this]() { this->tcmalloc_gc_thread(); },
             &_tcmalloc_gc_thread);
-    CHECK(st.ok()) << st.to_string();
+    CHECK(st.ok()) << st;
```

Comment:
Could we add another MACRO like CHECK(STATUS), IF STATUS not ok, just print statck error stack inside the marco.